### ISSUE TITLE
Newsletter: It is possible to send newsletters to recipients which have newsletterActive and/or newsletterConfirmed disabled

### ISSUE THREAD

**Issue by lukas-schnieper** [pimcore/pimcore#439]
### Problem
 It is possible to send a Newsletter, even if the recipients have newsletterActive = 0 or newsletterConfirmed = 0

### Reproduction:
Put an OR Filter into the Object-Filter (SQL). For Example:
`checkbox1 = 1 OR checkbox2 = 1`

The Code in send-newsletter.php will join this filter with the following result:
`(newsletterActive = 1 AND newsletterConfirmed = 1) AND checkbox1 = 1 OR checkbox2 = 1`

Which obviously sends the newsletter to all recipients which have `checkbox2 = 1`, even if the are not confirmed nor active.

### Solution:
Simply put "()" around your Object-Filter is a Solution, but a weak one. I would prefer that this is prevented in the code itself. So i propose adding these brackets in:

newsletter-send.php:38
`$conditions[] = "(".$newsletter->getObjectFilterSQL().")";`

So the new filter will be:
`(newsletterActive = 1 AND newsletterConfirmed = 1) AND (checkbox1 = 1 OR checkbox2 = 1)`

Which prevents sending newsletters to inactive recipients!

Regards
Lukas
